app/views/home/index.html.slim

```diff
@@ -55,59 +54,71 @@
         = "Tim Peacock"
         = ' / '
         = link_to "cc", 'https://creativecommons.org/licenses/by/2.0/'
+  button.carousel-control-prev[type="button" data-bs-target="#myCarousel" data-bs-slide="prev"]
+    span.carousel-control-prev-icon[aria-hidden="true"]
+    span.visually-hidden
+      | Previous
+  button.carousel-control-next[type="button" data-bs-target="#myCarousel" data-bs-slide="next"]
+    span.carousel-control-next-icon[aria-hidden="true"]
+    span.visually-hidden
+      | Next
 
 .container.homepage
-  .col-md-12 style="margin-bottom: 80px;"
-    .col-md-4 style="min-height: 160px;"
-      = image_tag "enterprise.png", class: 'img-circle', alt: 'Enterprise'
-      h2
-        = t('home.marketing.mooc.header_html')
-      p
-        = t('home.marketing.mooc.paragraph')
-    .col-md-4 style="min-height: 160px;"
-      = image_tag "education.png", class: 'img-circle', alt: 'Education'
-      h2
-        = t('home.marketing.universities.header_html')
-      p
-        = t('home.marketing.universities.paragraph')
-    .col-md-4 style="min-height: 160px;"
-      = image_tag "home.png", class: 'img-circle', alt: 'Home'
-      h2
-        = t('home.marketing.schools.header_html')
-      p
-        = t('home.marketing.schools.paragraph')
+  .row.mb-3
+    .col-md-4.col-sm-12 style="min-height: 160px;"
+      .card.text-dark.bg-light.m-1
+        = image_tag "enterprise.png", class: 'img-circle card-img-top pt-1', alt: 'Enterprise'
```

Comment:
The string used as alt text should be correctly localized and added to `config/locales/en.yml` (even if only used for screen readers).